docs/data.rst

```diff
@@ -114,6 +114,9 @@ In an Output, :meth:`Instance.set_dictionary` receives a dictionary with all or
 some of the Output type members, and in an Input, :meth:`SampleIterator.get_dictionary`
 retrieves all the members.
 
+It is also possible to provide a member_name to :meth:`SampleIterator.get_dictionary` to obtain
```

Comment:
Use `` around member_name.

"a dictionary containing the fields of that nested member only."